docs/elasticsearch_plugins.rst

```diff
@@ -122,16 +125,19 @@ To make this work, you need to manually edit Rally's configuration file in ``~/.
 
     plugin.my-plugin.remote.repo.url = git@github.com:example-org/my-plugin.git
     plugin.my-plugin.src.dir = /path/to/your/plugin/sources
-    plugin.my-plugin.build.task = :my-plugin:plugin:assemble
+    plugin.my-plugin.build.command = /usr/local/bin/gradle :my-plugin:plugin:assemble
     plugin.my-plugin.build.artifact.subdir = build/distributions
 
 Let's discuss these properties one by one:
 
 * ``plugin.my-plugin.remote.repo.url`` (optional): This is needed to let Rally checkout the source code of the plugin. If this is a private repo, credentials need to be setup properly. If the source code is already locally available you may not need to define this property. The remote's name is assumed to be "origin" and this is not configurable. Also, only git is supported as revision control system.
 * ``plugin.my-plugin.src.dir`` (mandatory): This is the absolute directory to which the source code will be checked out.
-* ``plugin.my-plugin.build.task`` (mandatory): The Gradle task to run in order to build the plugin artifact. This command is run from the plugin project's root directory.
+* ``plugin.my-plugin.build.command`` (mandatory): The full Gradle command to run in order to build the plugin artifact. This command is run from the plugin project's root directory.
```

Comment:
Actually, it does not have to be a Gradle command (E.g. Maven would also work here). But it is up to the user to make this work (build tool needs to be on the path and such things).